playbooks/roles/openconnect/templates/instructions.md.j2

```diff
@@ -41,23 +55,24 @@ OpenConnect is an extremely high-performance and lightweight VPN server that als
 
 <a name="osx"></a>
 ### OS X ###
+
 1. Install [Homebrew](https://brew.sh/), if you haven't already.
 1. Install OpenConnect using Homebrew:
 
    `brew install openconnect`
 
    * If installing Homebrew is not an option, you can also download and compile the [OpenConnect source code](/mirror/#openconnect).
-1. Download the CA and client certificate files:
-   * [ca-cert.pem](/openconnect/ca-cert.pem)
-   * [client.p12](/openconnect/client.p12)
-1. Place the downloaded files into a separate folder (e.g. `{{ streisand_server_name }}-openconnect`), open your Terminal, and `cd` to that directory.
+1. Download the [ca-cert.pem](/openconnect/ca-cert.pem) file, and a [client certificate file](#clientcerts) from the list above.
```

Comment:
Instead of using the file name here (`ca-cert.pem`) I think you should use the same term as the docs (`server certificate file`). That matches the client certificate file link.